docs/content/docs/operations/ingress.md

```diff
@@ -24,4 +24,36 @@ specific language governing permissions and limitations
 under the License.
 -->
 
+# Accessing Flink’s Web UI
+
+The Flink Kubernetes Operator, by default, does not change the way the native kubernetes integration [exposes](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/resource-providers/native_kubernetes/#accessing-flinks-web-ui) the Flink Web UI.
+
 # Ingress
+The Operator also supports creating an optional Ingress entry for the Web UI. Ingress generation can be turned on by defining the `ingressDomain` variable in the FlinkDeployment:
+
+```yaml
+apiVersion: flink.apache.org/v1alpha1
+kind: FlinkDeployment
+metadata:
+  namespace: default
+  name: basic-ingress
+spec:
+  image: flink:1.14.3
+  flinkVersion: v1_14
+  ingressDomain: flink.k8s.io
+  ...
+```
+
+The Operator takes the `ingressDomain`, `name` and `namespace` values from the deployment and creates an Ingress using the template `{{name}}.{{namespace}}.{{ingressDomain}}` to generate the host. This requires that anything `*.flink.k8s.io` should be routed to the Ingress controller on the Kubernetes cluster.
```

Comment:
Please add the following link to `Ingress` in this sentence:

https://kubernetes.io/docs/concepts/services-networking/ingress/

Reply:
Added, thanks @mbalassi